libs/auth/cache/file.go

```diff
@@ -73,6 +73,29 @@ func (c *FileTokenCache) Lookup(key string) (*oauth2.Token, error) {
 	return t, nil
 }
 
+func (c *FileTokenCache) DeleteKey(key string) error {
+	err := c.load()
+	if errors.Is(err, fs.ErrNotExist) {
+		return ErrNotConfigured
+	} else if err != nil {
+		return fmt.Errorf("load: %w", err)
+	}
+	c.Version = tokenCacheVersion
```

Comment:
Why set this?

Reply:
No reason to do it, great catch! Fixed in commit 89d3b1a4df8c3f5a4446169c5182f92e8ceb6801